asyncio_run_in_process/_child_trio.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import os
+import signal
+import sys
+from typing import (
+    Any,
+    AsyncIterator,
+    Awaitable,
+    BinaryIO,
+    Callable,
+    Sequence,
+)
+
+import trio
+import trio_typing
+
+from ._utils import (
+    pickle_value,
+    receive_pickled_value,
+)
+from .state import (
+    State,
+    update_state,
+    update_state_finished,
+    update_state_initialized,
+)
+from .typing import (
+    TReturn,
+)
+
+#
+# CLI invocation for subprocesses
+#
+parser = argparse.ArgumentParser(description="trio-run-in-process")
+parser.add_argument(
+    "--parent-pid", type=int, required=True, help="The PID of the parent process"
+)
+parser.add_argument(
+    "--fd-read",
+    type=int,
+    required=True,
+    help=(
+        "The file descriptor that the child process can use to read data that "
+        "has been written by the parent process"
+    ),
+)
+parser.add_argument(
+    "--fd-write",
+    type=int,
+    required=True,
+    help=(
+        "The file descriptor that the child process can use for writing data "
+        "meant to be read by the parent process"
+    ),
+)
```

Comment:
Thoughts on DRY-ing this and putting the version of this from `_child.py` somewhere reusable:

https://github.com/ethereum/asyncio-run-in-process/blob/9ff64a1c6bc27fb8a4be27d9a2a2b97003b4c627/asyncio_run_in_process/_child.py#L224-L245

Reply:
How about keeping it in `_child.py` and importing from there in `_child_trio.py`? I don't see it being used anywhere else, so probably don't make sense to have yet another module just for it